### ISSUE TITLE
[Suggestion] Focus: Flare

### ISSUE THREAD

**Issue by LincT** [TheCodex6824/ThaumicAugmentation#342]
Another vanilla interaction as a focus idea. 
tldr: focus that spawns fireworks and allows certain social interactions.

Power could be how far it travels.
it could have options on creation that affect the complexity such as the ball type as defined here:
https://minecraft.fandom.com/wiki/Firework_Star#Effects
perhaps even allowing a mode key toggle to cycle through the options or the colors.

Primary use case:
you and another friend are on a server trying to locate eachother, one of you can fire a flare to signal for help.

Unlike normal fireworks I would suggest the "Self" target simply sets off a firework from ones location, or causes an explosion with the burst dealing damage to the player, with projectile allowing one to fire a rocket, and touch/bolt allowing firework explosion from the target. This is to keep the rest of the mod balance without allowing a vis powered flight boost negating the value of the harness (and because the air focus already allows one to boost while flying, at a damage penalty to the person using the air focus)
